concent_api/concent_api/system_check.py

```diff
@@ -812,3 +829,15 @@ def check_additional_verification_call_time(app_configs: None=None, **kwargs: An
     if not isinstance(settings.ADDITIONAL_VERIFICATION_CALL_TIME, int) or settings.ADDITIONAL_VERIFICATION_CALL_TIME < 0:
         return [create_error_62_additional_verification_call_time_has_wrong_value()]
     return errors
+
+
+@register()
+def check_sci_callback_retries(app_configs: None=None, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    if 'concent-api' in settings.CONCENT_FEATURES:
+        if not hasattr(settings, 'SCI_CALLBACK_RETRIES_TIME'):
+            return [create_error_63_sci_callback_retries_is_not_set()]
+        if not isinstance(settings.SCI_CALLBACK_RETRIES_TIME, int) or settings.SCI_CALLBACK_RETRIES_TIME < 0 or \
```

Comment:
Please fix styling in breaking this if into lines.

Reply:
Could be each condition in separate line, more readable, but this is still OK ;)